graylog2-web-interface/src/views/logic/valueactions/createEventDefinition/CreateEventDefinitionModal.tsx

```diff
@@ -0,0 +1,230 @@
+    });
+  }, [modalData]);
+  const searchLabels = useMemo<{ [name: string]: JSX.Element }>(() => {
+    return mapValues(pick(modalData, searchGroup), (value, key: ItemKey) => (
+      <CheckboxLabel itemKey={key} value={value} />
+    ));
+  }, [modalData]);
+
+  const restLabels = useMemo<{ [name: string]: JSX.Element }>(() => {
+    return mapValues(omit(modalData, [...aggregationGroup, ...searchGroup]), (value, key: ItemKey) => (
+      <CheckboxLabel itemKey={key} value={value} />
+    ));
+  }, [modalData]);
+
+  const eventDefinitionCreationUrl = useMemo(() => {
+    return `${Routes.ALERTS.DEFINITIONS.CREATE}?step=condition&config=${JSON.stringify(urlConfig)}`;
+  }, [urlConfig]);
+
+  const strategyAvailabilities = useMemo<{[name in StrategyId]: boolean}>(() => {
+    return ({
+      ALL: true,
+      ROW: !!mappedData?.rowValuePath?.length,
+      COL: !!mappedData?.columnValuePath?.length,
+      CUSTOM: true,
+      EXACT: true,
+    });
+  }, [mappedData?.columnValuePath?.length, mappedData?.rowValuePath?.length]);
+
+  return (
+    <Modal onHide={onClose} show={show}>
+      <Modal.Header closeButton>
+        <Modal.Title>Add values to Event Definition</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <RadioSection strategyAvailabilities={strategyAvailabilities} strategy={strategy} onChange={onStrategyChange} />
+        <Button bsStyle="link" className="btn-text" bsSize="xsmall" onClick={toggleDetailsOpen}>
+          <Icon name={`caret-${showDetails ? 'down' : 'right'}`} />&nbsp;
+          {showDetails ? 'Hide strategy details' : 'Show strategy details'}
+        </Button>
+        {
+          showDetails && (
+          <div>
+            {!isEmpty(aggregationChecks) && (
+            <CheckBoxGroup onChange={onCheckboxChange}
+                           groupLabel="Aggregation"
+                           checked={aggregationChecks}
+                           labels={aggregationLabels} />
+            )}
+            {!isEmpty(searchChecks) && (
+            <CheckBoxGroup onChange={onCheckboxChange}
+                           groupLabel="Search query"
+                           checked={searchChecks}
+                           labels={searchLabels} />
+            )}
+            {
+              Object.entries(restChecks).map(([key, isChecked]) => (
+                <Checkbox checked={isChecked} onChange={() => onCheckboxChange({ [key]: !isChecked })}>
+                  {restLabels[key]}
+                </Checkbox>
+              ))
+            }
+          </div>
+          )
+        }
+      </Modal.Body>
+      <Modal.Footer>
+        {/*
+        <Button type="button" onClick={onClose}>
+          Cancel
+        </Button>
+        <Button bsStyle="primary" to="/search">
+          Go to event definition
+        </Button>
+        */}
+        <Link onClick={onClose} to={eventDefinitionCreationUrl} target="_blank">Go to event definition</Link>
```

Comment:
"Go to event definition" sounds like the event definition already exists. Maybe let's name it "Continue configuration"?

Reply:
I guess you already tried to use a button here?

Reply:
What is the benfit of using redux here? Have you considered using a form here with a classic submit button?

With formik you could easily add validation, etc. On submit you could call `window.open` to open the event definition wizard.

Reply:
> What is the benfit of using redux here? Have you considered using a form here with a classic submit button?
> 
> With formik you could easily add validation, etc. On submit you could call `window.open` to open the event definition wizard.

It's no redux. It react `useReducer` hook. The benefit is that in some cases we are changing several states simultaneously. 
In this case, we don't really need validation and form. Also, we have many dependencies on what to change in the form depending on what was changed. That more straightforward implementation without Formik. Especially when we don't need form here